Client/Ecosia/UI/MyImpact/NewsController.swift

```diff
@@ -133,8 +142,19 @@ final class NewsController: UIViewController, UICollectionViewDelegate, UICollec
     }
 }
 
+private extension UICollectionView {
+    func align(header: NewsSubHeader?) {
+        let margin = (bounds.width - ecosiaHomeMaxWidth) / 2
```

Comment:
Instead of adding a cap every time we use this computed property I opted to have inside the implementation, so that it is safe to use, refer to [here](https://github.com/ecosia/ios-browser/blob/163037f843c4a3db573019e2a741027eaeec2d3a/Client/Ecosia/UI/MyImpact/UICollectionView%2BmaxWidth.swift#L10).

TL;DR `ecosiaHomeMaxWidth` can't be greater than `bounds.width`.